mpd/config.json

```
@@ -32,12 +32,16 @@
   },
   "options": {
     "volume_normalization": false,
-    "httpd_output": false
+    "httpd_output": false,
+    "music_folder": "/share/mpd/music",
```

```suggestion
    "media_folder": "/media/mpd/media",
```

Since it can also include all types of meda. Please change all other occurences as well